wad/lumps/language.lmp

```diff
@@ -676,7 +676,7 @@ OB_MPBFG_BOOM = "%o was splintered by %k's BFG.";
 OB_MPBFG_SPLASH = "%o couldn't hide from %k's BFG.";
 OB_MPTELEFRAG = "%o was telefragged by %k.";
 OB_RAILGUN = "%o was railed by %k.";
-OB_KILLED = "%o killed %k.";
+OB_KILLED = "%k killed %o.";
```

Comment:
Looks like this is the only obit with "<killer> killed <victim>" rather than "<victim> was killed by <killer>." Could this be changed?

Reply:
Looks like github thought my comment was html and stripped it. I meant [killer] killed [victim] rather than [victim] was killed by [killer].